src/components/review/ReviewPersonal.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import CardHeader from '@mui/material/CardHeader';
+import { Avatar } from '@mui/material';
+import stringToAvatar from '../../helpers/string-to-avatar';
+export default function ReviewPersonal({reviewDate,reviewer}) {
+  return (
+    <>
+     <CardHeader
+        avatar={
+          <Avatar {...stringToAvatar(`${reviewer.firstName} ${reviewer.lastName}`)}/>
+        }
+        action={new Date((reviewDate)).toDateString()}
+        title={`${reviewer.firstName} ${reviewer.lastName}` }
```

Comment:
The fullname is used twice, perhaps you could define the variable fullname and use it on both places